docs/src/content/docs/ru/environmental-impact.md

```diff
@@ -0,0 +1,149 @@
+
+### Изображения
+
+Хоть Starlight и предлагает легкие страницы по-умолчанию, изображения, которые вы добавляете на страницы документации, могут быстро увеличивать вес вашей страницы.
+Starlight использует [оптимизировацию ресурсов][assets] Astro для оптимизации локальных изображений в ваших файлах Markdown и MDX.
+
+### UI компоненты
+
+Компоненты, на UI фреймворках, как React или Vue, могут легко добавлять большие объемы JavaScript на страницу.
+Поскольку Starlight основан на Astro, эти компоненты, по умолчанию **не загружают клиентский JavaScript** благодаря [Astro Островам][islands].
+
+### Кеширование
+
+Кэширование управляет тем, как долго браузер хранит и повторно использует данные, которые он уже загрузил.
+Хорошая стратегия кэширования гарантирует, что пользователь получает новое содержание как можно быстрее,
+когда оно меняется, но также избегает бесполезной повторной загрузки одного и того же содержания снова и снова, когда оно не изменилось.
+
+Самым распространенным способом настройки кэширования является использование [HTTP-заголовка `Cache-Control`][cache].
+При использовании Starlight вы можете установить длительное время кэширования для всего, что находится в каталоге `/_astro/`.
+Этот каталог содержит CSS, JavaScript и другие ресурсы, которые можно безопасно кэшировать навсегда, что позволяет снизить избыточные загрузки:
+
+```
+Cache-Control: public, max-age=604800, immutable
+```
+
+Как настроить кэширование зависит от вашего веб-хоста. Например, Vercel автоматически применяет эту стратегию кэширования без необходимости настройки,
+в то же время вы можете установить [заголовки для Netlify][ntl-headers], добавив файл `public/_headers` в ваш проект:
+
+```
+/_astro/*
+  Cache-Control: public
+  Cache-Control: max-age=604800
+  Cache-Control: immutable
+```
+
+[cache]: https://csswizardry.com/2019/03/cache-control-for-civilians/
+[ntl-headers]: https://docs.netlify.com/routing/headers/
+
+## Потребление энергии
+
+То, как реализована веб-страница может влиять на потребление энергии при ее запуске на устройстве пользователя. 
+За счет минимального использования JavaScript, Starlight снижает объем вычислительных ресурсов, необходимых телефону,
+планшету или компьютеру пользователя для загрузки и отображения страниц.
+
+Будьте внимательны при добавлении функций, таких как скрипты отслеживания аналитики или контент, зависящий от JavaScript,
+например, встроенные видео, так как они могут увеличить энергопотребление страницы. 
+Если вам необходима аналитика, рассмотрите выбор легковесного варианта, 
+такого как [Cabin][cabin], [Fathom][fathom] или [Plausible][plausible]. 
+Встроенные видео, такие как YouTube и Vimeo, можно улучшить, ожидая [взаимодействие пользователя для загрузки видео][lazy-video].
+Пакеты, такие как [`astro-embed`][embed], могут помочь с часто используемыми сервисами.
+
+:::tip[Знаете ли вы?]
+Разбор и компиляция JavaScript являются одной из самых ресурсоемких задач, которые браузерам приходится выполнять."
```

Comment:
I think the quotation mark at the end here is a typo, right?

```suggestion
Разбор и компиляция JavaScript являются одной из самых ресурсоемких задач, которые браузерам приходится выполнять.
```

Reply:
oops. Thanks